src/base/path.cpp

```diff
@@ -288,7 +295,9 @@ Path operator/(const Path &lhs, const Path &rhs)
     if (lhs.isEmpty())
         return rhs;
 
-    return Path(lhs.m_pathStr + QLatin1Char('/') + rhs.m_pathStr);
+    return Path::createUnchecked(rhs.isAbsolute()
+        ? QString(lhs.m_pathStr + rhs.m_pathStr)
```

Comment:
Having absolute path at right hand is invalid case. We cannot prevent such misuse except by throwing an exception. But this is not our way. So we're leaving it up to the caller. Here we can just add an assertion.

Reply:
There are such usage in webui. I'll leave it to others.

Reply:
> There are such usage in webui. I'll leave it to others.

Can you tell me where exactly?

Reply:
> For example, rhs may have `/scripts/file.css` and lhs may be `/private`.

This seems to be an exceptional case. From the point of view of server logic, `/scripts/file.css` doesn't refer to absolute path, but to the path that is relative against "site root" folder. We can either leave everything as it is, or make the requirement suggested above, but handle this case explicitly:
```c++
const auto path = Path("/private") / Path("/").relativePathOf("/scripts/file.css");
```

Reply:
>but handle this case explicitly:

I don't feel strongly for it, maybe keep it as is for now.